docs/snippets/_private-operator-release-matrix.mdx

```diff
@@ -1,3 +1,4 @@
 | Release | Version | Date |  Release Notes | AWS Version | GCP Download | Azure Download |
 | ------- | ------  | ------ | ------ | ------ | ------ | ------ | 
-| Q1 2024 | 5.26.19  | Feburary 13 2024 | [v5.26.19-56899dc0d7](https://github.com/IABTechLab/uid2-operator/releases/tag/v5.26.19-56899dc0d7) | 5.26.19-56899dc0d7 | [5.26.19-56899dc0d7 GCP ZIP](https://github.com/IABTechLab/uid2-operator/releases/download/v5.26.19-56899dc0d7/uid2-operator-deployment-artifacts-5.26.19-56899dc0d7-gcp-oidc.zip) | [5.26.19-56899dc0d7 Azure ZIP](https://github.com/IABTechLab/uid2-operator/releases/download/v5.26.19-56899dc0d7/uid2-operator-deployment-artifacts-5.26.19-56899dc0d7-azure-cc.zip) |
+| Q1 2024 | 5.26.19 | Feburary 13 2024 | [v5.26.19-56899dc0d7](https://github.com/IABTechLab/uid2-operator/releases/tag/v5.26.19-56899dc0d7) | 5.26.19-56899dc0d7 | [5.26.19-56899dc0d7 GCP ZIP](https://github.com/IABTechLab/uid2-operator/releases/download/v5.26.19-56899dc0d7/uid2-operator-deployment-artifacts-5.26.19-56899dc0d7-gcp-oidc.zip) | [5.26.19-56899dc0d7 Azure ZIP](https://github.com/IABTechLab/uid2-operator/releases/download/v5.26.19-56899dc0d7/uid2-operator-deployment-artifacts-5.26.19-56899dc0d7-azure-cc.zip) |
```

Comment:
February 13, 2024
(fix typo plus with the comma is more usual American presentation)